tests/helpers/default-packager.js

```diff
@@ -54,7 +54,7 @@ const DEFAULT_NODE_MODULES = {
       `,
     'package.json': JSON.stringify({
       name: 'ember-source',
-      version: '~3.0.0-beta.1',
+      version: '3.0.0-beta.1',
```

Comment:
I corrected this because the version property on an addon's package.json should not have ~ or ^.